CHANGES.md

```diff
@@ -1,3 +1,9 @@
+1.0.2
+=====
+
+* [#97](https://github.com/serokell/universum/issues/97):
+  Add `ToPairs` type family with the ability to have list of pairs.
```

Comment:
`ToPairs` is not a type family actually, it's a type class with associated type families.